esp/esp/program/modules/handlers/volunteermanage.py

```diff
@@ -73,7 +75,14 @@ def volunteering(self, request, tl, one, two, module, extra, prog):
             response['Content-Disposition'] = 'attachment; filename=volunteers.csv'
             return response
 
-        if 'op' in request.GET:
+        elif extra == "request_import":
```

Comment:
Can we just make `volunteer_import` an `aux_call` instead of using `extra`?  I know `csv` already does this but I think that's ugly too; this function is complicated enough when it's not trying to render several mostly-different pages.